engines/dfc_provider/app/controllers/dfc_provider/api/persons_controller.rb

```diff
@@ -4,7 +4,7 @@
 module DfcProvider
   module Api
     class PersonsController < DfcProvider::Api::BaseController
-      skip_before_action :check_enterprise
+      skip_before_action :set_enterprise
```

Comment:
Skipping is a sign that the before action shouldn't be activated in the BaseController but called where needed. Less automation is simpler to understand.

Reply:
I have reverse it by just applying the filter when required, and still kept it inside the BaseController. Or do you think we should use a mixin for this? It is a bit too much at this step, logic is really simple, right?